src/app/shared/libraries/libraries.module.ts

```diff
@@ -1,17 +1,17 @@
-import {CommonModule} from '@angular/common';
-import {NgModule} from '@angular/core';
-import {AccordionComponent} from '@app/shared/libraries/accordion/accordion.component';
-import {AccordionItemComponent} from '@app/shared/libraries/accordion-item/accordion-item.component';
+import { CommonModule } from '@angular/common';
+import { CUSTOM_ELEMENTS_SCHEMA, NgModule } from '@angular/core';
+import { AccordionComponent } from '@app/shared/libraries/accordion/accordion.component';
+import { AccordionItemComponent } from '@app/shared/libraries/accordion-item/accordion-item.component';
 
 export const PIPES_AND_COMPONENTS = [
   AccordionComponent,
   AccordionItemComponent
 ];
 
 @NgModule({
-  imports: [ CommonModule ],
+  imports: [CommonModule],
   declarations: PIPES_AND_COMPONENTS,
-  exports: PIPES_AND_COMPONENTS
+  exports: PIPES_AND_COMPONENTS,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA]
```

Comment:
` schemas: [CUSTOM_ELEMENTS_SCHEMA]` we wont need this here @irina005  @anatolmanoila. likewise in the import